cryodrgn/commands/eval_vol.py

```diff
@@ -27,6 +27,7 @@ def add_args(parser):
     parser.add_argument('-c', '--config', metavar='PKL', required=True, help='CryoDRGN config.pkl file')
     parser.add_argument('-o', type=os.path.abspath, required=True, help='Output .mrc or directory')
     parser.add_argument('--prefix', default='vol_', help='Prefix when writing out multiple .mrc files (default: %(default)s)')
+    parser.add_argument('--device', type=int, default=0, help='Optionally specify CUDA device')
```

Comment:
Since earlier code was calling this using `CUDA_VISIBLE_DEVICES=n cryodrgn eval_vols` (which would have worked on *nix/macOS platforms anyway), I added a `--device` argument to this function to emulate invocation with/without cuda. To maintain backward compatibility it uses cuda device `0` by default (and checks to see if cuda is indeed available at runtime so it'll still work if in the `cpu` case).